src/ServiceControl/Bootstrapper.cs

```diff
@@ -56,7 +50,7 @@ public Bootstrapper(Settings settings, BusConfiguration configuration)
         private void Initialize()
         {
             var loggingSettings = new LoggingSettings(settings.ServiceName);
-            ConfigureLogging(loggingSettings);
+            RecordStartup(loggingSettings);
```

Comment:
Yes but we now configure the logging long before referencing the `Settings` class [here](https://github.com/Particular/ServiceControl/pull/981/files#diff-a98df16b3668f42c86d0ab2da8be83b2R24)